### DISCUSSION TITLE
Would Anybody like to fight bots?

### DISCUSSION THREAD

**Discussion by Vanny-Chuck** [SebLague/Chess-Challenge#128] [Ideas]
We could perhaps send each other our code to put in EvilBot then have them fight in the background! At the end we could have screenshots of scores to find the best bot!

**Comment by KaizNike** [comment]
I like the idea! There's always the risk of code swipers, but I think it would be fun to test our code against one another before we turn in.

**Comment by draeg82** [comment]
I've made an awful bot which should get itself in the position to allow a good bot to test it's end game.  if that's useful.

**Reply by Vanny-Chuck** [reply]
Cool if you would like to share the code that would be awesome!

**Reply by carsonetb** [reply]
Oh yeah that would be cool.

**Comment by MinekPo1** [comment]
I'm willing to play via discord/matrix as suggested in #77 by @manfromfms
